mattermost-plugin/webapp/src/components/cloudUpgradeNudge/cloudUpgradeNudge.tsx

```diff
@@ -8,7 +8,7 @@ const PostTypeCloudUpgradeNudge = (props: {post: Post}): JSX.Element => {
     const ctaHandler = (e: React.MouseEvent) => {
         e.preventDefault()
         const windowAny = (window as any)
-        windowAny?.openPricingModal()()
+        windowAny?.openPricingModal()({trackingLocation: 'boards > click_view_upgrade_options_nudge'})
```

Comment:
@sri-byte Would there be any issues introducing spaces and `>` here? It seems that the convention has been snake_case - maybe we can keep it consistent?

Reply:
Hey @sri-byte , gentle ping here

Reply:
I am not sure, is it possible we can view this in the testing environment